landoapi/models/treestatus.py

```diff
@@ -234,9 +203,9 @@ class StatusChangeTree(Base):
         db.String(64), db.ForeignKey(Tree.tree), nullable=False, index=True
     )
 
-    # A JSON encoded string containing the previous state of the tree before
+    # A JSON object containing the previous state of the tree before
     # applying this change.
-    last_state = db.Column(db.Text, nullable=False)
+    last_state = db.Column(JSONB, nullable=False)
```

Comment:
Does this need a default?

Reply:
No, a missing status is a programming error.